docs/basicuse.rst

```diff
@@ -1,7 +1,7 @@
 Basic Use
 ---------
 
-.. code-block:: bash
+.. code-block:: guess
```

Comment:
PS ... I spent some time trying to understand why it couldn't parse this one section and ultimately gave up,. The highlighting here is just not that important.